src/components/Error/Empty.jsx

```diff
@@ -27,7 +27,7 @@ const EmptyCanvas = ({ type, canUpload, localeKey, hasTextMobileVersion }) => {
   const { isMobile } = useBreakpoints()
 
   let iconToShow = EmptyIcon[type]
-  if (isTwakeTheme() && type !== 'trash') {
+  if (isTwakeTheme()) {
```

Comment:
I meant to use `isTwakeTheme()` only **somehow** 🤔 so be changing the code if necessary... we have the same result here than before?